src/sider/dialogmoter/components/motehistorikk/MoteSvarHistorikk.tsx

```diff
@@ -1,11 +1,11 @@
 import { DialogmoteDTO } from "@/data/dialogmote/types/dialogmoteTypes";
-import { FortidenImage } from "../../../../../img/ImageComponents";
-import { Accordion, BodyLong, BodyShort, Box, Heading } from "@navikt/ds-react";
+import { Accordion, BodyShort, Box } from "@navikt/ds-react";
 import React from "react";
 import MoteSvarHistorikkEvent from "@/sider/dialogmoter/components/motehistorikk/MoteSvarHistorikkEvent";
+import DialogmoteHistorikkHeader from "@/sider/dialogmoter/components/motehistorikk/DialogmoteHistorikkHeader";
 
 const texts = {
-  header: "Møtesvarhistorikk",
+  header: "Svar på dialogmøte historikk",
```

Comment:
Er `Historikk dialogmøtesvar` bedre? Føles litt som at det blir orddelingsfeil her, at det liksom er `"Svar på dialogmøte"-historikk`.
Men det er kanskje en bevisst greie fra Peter at vi kaller det "Svar på dialogmøte" i stedet for "Dialogmøtesvar"? 🤔 Tenker uansett vi kan dytte dette ut sånn som det står nå 👍🏼